language.go

```diff
@@ -18,7 +18,8 @@ import (
 	"strings"
 	"unicode"
 
-	surface "github.com/googleapis/gnostic/surface"
+	gnostic_grpc "github.com/googleapis/gnostic-grpc/generator"
+	surface_v1 "github.com/googleapis/gnostic/surface"
```

Comment:
is the `_v1` suffix necessary? I would avoid using it if possible.

Reply:
I may have introduced that pattern somewhere else, but agree with @noahdietz here, "surface" seems fine.

Reply:
Done